CONTRIBUTING.md

```diff
@@ -1,3 +1,15 @@
 # Contributing
 
 We love pull requests from everyone. By participating in this project, you agree to abide by the Web Matters [code of conduct](http://www.web-matters.co.uk/code-of-conduct.html).
+
+## Process
+
+The site is run by GitHub from `master` so do not directly commit to `master`.
```

Comment:
> The site is served by GitHub Pages from `master`
Again, just a minor suggestion to make the workflow slightly clearer to a layperson.